gr-vocoder/python/vocoder/qa_freedv.py

```diff
@@ -0,0 +1,70 @@
+#!/ usr / bin / env python
```

Comment:
Both the shebang and the copyright year seem wrong. The shebang can go, right? This doesn't get manually executed, IIRC.

Reply:
well, it's usual that our QA codes have that, IIRC; I'll check (I personally regularly call qa_foobar_stuff.sh manually, but that calls python3 qa_foobar_stuff.py)

Reply:
Don't know how that happened, though, and that kinda worries me